packages/ui/src/components/cell/cell.native.tsx

```diff
@@ -67,19 +75,21 @@ export function Cell({ title, subtitle, onPress, Icon, ...props }: CellProps) {
       justifyContent="space-between"
       alignItems="center"
     >
-      <Box flexDirection="row" gap="4">
-        <Box flexDirection="row" p="2" bg={getIconBackgroundColor(_variant)} borderRadius="round">
-          <Icon color={theme.colors[getIconColor(_variant)]} />
-        </Box>
-        <Box flexDirection="column" justifyContent="center">
-          <Text variant="label02">{title}</Text>
-          {subtitle && (
-            <Text color="ink.text-subdued" variant="label03">
-              {subtitle}
-            </Text>
-          )}
-        </Box>
-      </Box>
+      {/* <ItemLayout
```

Comment:
Tbh, I'm a little confused about our use of these three components vs the extension. In the extension, we use `Pressable` as a wrapper to manage the different states, and we never put `ItemLayout` or `Flag` in that component directly. Those layout components are used as needed with Pressable. In RN, we already have `TouchableOpacity` and `Pressable` to use, so it seems like `Cell` is just adding some more functionality to those and shouldn't actually layout anything?

Reply:
I am pretty sure my `Accordian` component is also considered to be a `Cell` by the design team? See link:
https://www.figma.com/design/FlmF7ropcijOb7iVSDqNn0/%F0%9F%9F%AA-Web-App-Explorations-2024-H2?node-id=14270-180244&node-type=FRAME&t=8f2WXLEvxAMwQ3rI-0

Reply:
Thanks @fbwoolf . I am confused too so I think the best thing to do is just leave this `Cell` as it was without `ItemLayout` or `Flag`. 

It seems like over-engineering for now and we can refactor it in future if needed and maybe remove `ItemLayout` / `Flag` for native if they aren't used.

Reply:
@kyranjamie curious your thoughts here? My thought is we would want to refactor all the uses of Cell to utilize `ItemLayout` and `Flag` as need, rather than use those directly in `Cell`? Or, the `Cell` itself needs to handle the variants seen in the designs and decide which layout component to use? Rn, the `Cell` component really isn't what is show in the 4 variants asked for in the designs? Or, am I missing something?

Reply:
I can refactor `item-layout.native` to use the style we have in `Cell` now? Or else we can just keep it the way it is rather than importing `<Itemlayout` to one place. 
![Screenshot 2024-09-02 at 16 04 01](https://github.com/user-attachments/assets/82545460-095a-462c-84d0-afa979fcbcca)

I think the `Cell` can show the variants now, apart from the `radio` button one. 



For the last variant, what we have now works but we could also use `<TouchableOpacity>` and `<Flag reverse` to do the same thing